web/src/rest.ts

```diff
@@ -233,8 +233,13 @@ const dandiRest = new Vue({
     assetManifestURI(identifier: string, version: string) {
       return `${dandiApiRoot}dandisets/${identifier}/versions/${version}/assets/`;
     },
-    assetDownloadURI(identifier: string, version: string, uuid: string) {
-      return `${dandiApiRoot}assets/${uuid}/download/`;
+    assetDownloadURI(asset: Asset|AssetFile) {
+      // @ts-ignore next-line
+      const { asset_id, zarr } = asset;
+      if (zarr) {
+        return `https://dandiarchive.s3.amazonaws.com/zarr/${zarr}`;
```

Comment:
@dchiquito is there a way to retrieve this s3 url from the API instead? I *think* the `/zarr/{zarr_id}.zarr/{path}` endpoint will do it, but I get a 404 when I try to call it like this -
https://api.dandiarchive.org/api/zarr/21e4e2e9-9474-4568-8dee-ae9154316ddd.zarr/

Reply:
>  but I get a 404 when I try to call it like this -
https://api.dandiarchive.org/api/zarr/21e4e2e9-9474-4568-8dee-ae9154316ddd.zarr/

that's because ingest hasn't been completed for all of them. (see #1063 issue)

Reply:
Update: I was able to get this endpoint working in staging, it seems it doesn't actually provide what we need here https://api-staging.dandiarchive.org/api/zarr/94fdcde1-afa8-437f-8d7b-c75967295843.zarr/

Reply:
> i would suggest using the s3 http endpoint if possible, since it gets the api requests to a much more horizontally scalable infrastructure.

We actually already have an `fsspec`-compliant zarr API that makes this very easy (thanks @dchiquito for pointing me to that), so unless there are scalability concerns that I'm not aware of, I think it'd be better to use that. Otherwise, we would need to augment the asset path endpoint to also return s3 urls.

Reply:
the asset metadata has s3 urls in it. and perhaps that could be retrieved on click rather than when listing the asset. since a click will be a much more infrequent event, there is no need to add the return to the asset path endpoint.

i think we are already seeing ngff api limits from the data backup script.